### PR TITLE
refactor: rely on pnpm filter to run specific examples

### PR DESCRIPTION
**Pull request by tylersayshi**
could probably do similar with the website, but scoping this PR to just the examples seems like an appropriate start

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,14 +20,7 @@ pnpm run compile
...
 ```shell
-pnpm run examples:dev:01_template
-```
-
-or if you want to run a specific example:
-
-```shell
-cd examples/01_template
-pnpm run dev
+pnpm -F 01_template dev # to run examples/01_template in dev mode
```
Comment (dai-shi) [inline CONTRIBUTING.md:23]: This is so good!
Can you add an instruction for the `pnpm run examples:prd:01_template` equivalent?

Comment (tylersayshi) [reply CONTRIBUTING.md:23]: is this good?

```shell
pnpm -F 01_template dev # to run examples/01_template in dev mode
```

`build` and `start` can be run with the same pattern:

```shell
pnpm -F 01_template build
pnpm -F 01_template start
```

---

Or do you want to explicitly call out:

`pnpm -F 01_template build &&  pnpm -F 01_template start`

Comment (dai-shi) [reply CONTRIBUTING.md:23]: Hm, whichever is fine. I'd probably use some hack personally. So, just consider a better way for docs.

Comment (tylersayshi) [reply CONTRIBUTING.md:23]: updated to the above without the `&&` 

happy hacking haha

**Review by dai-shi — APPROVED** [review]
LGTM

I thought a drawback for me was autocompletion, but it seem like pnpm has the support: https://pnpm.io/8.x/completion

**Pull request merged**